### DISCUSSION TITLE
Identified malware in the 0.200 version

### DISCUSSION THREAD

**Discussion by adam19110** [seerge/g-helper#3517] [General]
Windows integrated virus scanner identified Trojan:Script/Sabsik.FL.A!ml virus in the 0.200 version ZIP. Is it safe, or malwared the new version?

**Comment by seerge** [comment]
@adam19110 hello, 

All versions are assembled directly on Github itself by [github actions](https://github.com/features/actions) from open sources (that you see). 
It's a false positive from windows defender that can happen at random.

https://www.virustotal.com/gui/url/c31b4592e0d66cae5952828d0ce6b27883d6f269b37348545d0f55bf790263e1
https://github.com/seerge/g-helper/wiki/FAQ#windows-defender-or-any-other-antivirus-marks-app-as-malware--virus

**Reply by adam19110** [reply]
Thank you for the answer and your hard work :) I wish from hungary merry christmas and happy new year :)

**Reply by seerge** [reply]
Same to you
